docs/assumptions.md

```diff
@@ -381,3 +385,44 @@ class Child extends Parent {
 ```
 
 </div>
+
+## Migrating from `@babel/preset-env`'s `"loose"` and `"spec"` modes
+
+`@babel/preset-env`'s `loose` option is equivalent to the following configuration:
+```json title="JSON"
+{
+  "presets": [
+    ["@babel/preset-env", { "exclude": ["transform-typeof-symbol"] }]
+  ],
+  "assumptions": {
+    "arrayLikeIsIterable": true,
+    "constantReexports": true,
+    "ignoreFunctionLength": true,
+    "ignoreToPrimitiveHint": true,
+    "mutableTemplateObject": true,
+    "noClassCalls": true,
+    "noDocumentAll": true,
+    "noObjectSuper": true,
+    "noUndeclaredVariablesCheck": true,
+    "objectRestNoSymbols": true,
+    "privateFieldsAsProperties": true,
+    "pureGetters": true,
+    "setClassMethods": true,
+    "setComputedProperties": true,
+    "setPublicClassFields": true,
+    "setSpreadProperties": true,
+    "skipForOfIteratorClosing": true,
+    "superIsCallableConstructor": true
```

Comment:
It's basically everything except for noIncompleteNsDetection and iterableIsArray.